mu_repo/__init__.py

```diff
@@ -87,6 +109,11 @@ def main(config_file='.mu_repo', args=None, config=None):
         Print(msg)
         return Status(msg, False)
 
+    if config_file == '.mu_repo' and not os.path.isfile(config_file):
```

Comment:
> Why limit config_file == '.mu_repo'? 

I started with `config_file=None`, but I was having a problem with a test and thought that was the cause, but later found it was not and forgot to change it back.

Changed it back to the much saner `config_file=None` default. :grin: